src/KurrentDB/ClusterVNodeHostedService.cs

```diff
@@ -355,6 +360,9 @@ public Task StopAsync(CancellationToken cancellationToken) =>
 		Node.StopAsync(cancellationToken: cancellationToken);
 
 	public void Dispose() {
+		if (_clusterNodeMutex is { IsAcquired: true })
```

Comment:
@timothycoleman added this release on dispose when mutex was acquired to allow cleaning it up for parallel tests.